homeassistant/components/roborock/scene.py

```diff
@@ -0,0 +1,64 @@
+"""Support for Roborock scene."""
+
+from __future__ import annotations
+
+import asyncio
+from typing import Any
+
+from homeassistant.components.scene import Scene as SceneEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RoborockConfigEntry
+from .coordinator import RoborockDataUpdateCoordinator
+from .entity import RoborockEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: RoborockConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up scene platform."""
+    scene_lists = await asyncio.gather(
+        *[coordinator.get_scenes() for coordinator in config_entry.runtime_data.v1],
+    )
+    async_add_entities(
+        RoborockSceneEntity(
+            coordinator,
+            EntityDescription(
+                key=str(scene.id),
+                name=scene.name,
+            ),
+        )
+        for coordinator, scenes in zip(
+            config_entry.runtime_data.v1, scene_lists, strict=False
```

Comment:
My understanding is that `strict=False` implies its ok for the lists to not have the same length. That should not happen. You added this which wasn't in the example i had -- can you remove or help me understand why this could possibly be OK? It would just introduce bugs if it were true, no?

Reply:
This was added by automatically by Ruff and I failed to notice it. I chnaged it to True